dlq_processor/Readme.md

```diff
@@ -23,6 +23,32 @@ The following AWS Lambda environment variables are supported in both the lambda
 SumoCWProcessDLQLambda supports one extra environment variable
 * NUM_OF_WORKERS(REQUIRED): It’s default value is 4. It controls the number of instances of SumoCWProcessDLQLambda to spawn if there is no failure in first attempt.It helps in faster processing of pending messages in dead letter queue.
 
+# Dynamic Metadata Fields
+
+The lambda supports dynamically overriding the _sourceName, _sourceHost and _sourceCategory per log message by setting `_sumo_metadata` within a json log.
+
+This can be useful when writing to CloudWatch Logs via a lambda function.
+
+For example:
+
+```
+exports.handler = (event, context, callback) => {
+
+    var serverIp = '123.123.123.123'
+
+    console.log(JSON.stringify({
+        'message': 'something happened..',
+        '_sumo_metadata': {
+            'category': 'prod/appa/console',
```

Comment:
I think the keys should be X-Sumo-Name, X-Sumo-Category, X-Sumo-Host to match what's set in generateHeaders here: https://github.com/himanshu219/sumologic-aws-lambda/blob/cf7fa4f5ef5fb86ebb5140f4741e852e405fe2c7/dlq_processor/sumo-dlq-function-utils/lib/sumologsclient.js

Reply:
they need to be same as in metadataMap variable defined in https://github.com/himanshu219/sumologic-aws-lambda/blob/cf7fa4f5ef5fb86ebb5140f4741e852e405fe2c7/dlq_processor/sumo-dlq-function-utils/lib/sumologsclient.js therefore changing them to sourceName and 
sourceHost